cmd/skaffold/app/cmd/commands_test.go

```diff
@@ -105,6 +105,11 @@ func TestNewCmdWithFlags(t *testing.T) {
 	testutil.CheckDeepEqual(t, "usage", flags["test"].Usage)
 }
 
+func TestNewCmdWithHouseKeepingMessages(t *testing.T) {
```

Comment:
this will ensure if the annotations are added. But this test won't protect us against a regression introduced if someone removed this condition
https://github.com/GoogleContainerTools/skaffold/blob/90530d501a96264c6790d21f50b2972d5d4e36d0/cmd/skaffold/app/cmd/cmd.go#L92


Can you please add a test, where we make sure if output does not contain the messages
```
// Return showDisplayprompt to be always be true.
t.Override(&config.ShouldDisplayPrompt, true)
var buf bytes.Buffer

cmd := NewCmd("").ExactArgs(1, func(ctx context.Context, out io.Writer, args []string) error {
		fmt.Fprintf(out, "test output: %v\n", args)
		return nil
	})
cmd.SetOutput(&buf)
// Make sure the display prompt is not shown
```

Reply:
I think this will be easier to test in an integration test 🤷🏾‍♂️

Reply:
@gsquared94 agreed, I think it would be more effective as well

Reply:
added integration tests